utils/get_licenses_from_ort.py

```diff
@@ -94,17 +102,28 @@ def __str__(self):
                     else:
                         final_licenses = [license]
                     for license in final_licenses:
+                        package_license = PackageLicense(
+                            package["id"], ort_result.name, license
+                        )
                         if license not in APPROVED_LICENSES:
-                            unknown_licenses.append(
-                                PackageLicense(package["id"], ort_result.name, license)
-                            )
+                            unknown_licenses.append(package_license)
+                        else:
+                            final_packages.append(package_license)
                         all_licenses_set.add(license)
             except Exception:
                 print(
                     f"Received error for package {package} used by {ort_result.name}\n Found license={license}"
                 )
                 raise
 
+package_list_file_path = f"{SCRIPT_PATH}/final_package_list.txt"
```

Comment:
I think it would be better to add date to the file, it'll help tracking and debugging in case of issues, and avoid collision in case of changes to timing.

Reply:
we add the date on the ort workflow